plugins/services/src/js/containers/tasks/__tests__/TaskTable-test.js

```diff
@@ -57,19 +55,6 @@ describe("TaskTable", function() {
       ];
       expect(this.taskTable.getDisabledItemsMap(tasks)).toEqual({ "2": true });
     });
-
```

Comment:
Let's not drop this test. Let's change it so it asserts we DID pass the `uniqueProperty` of value `"id"` to the CheckboxTable.